hack/arktos-cni.rc

```diff
@@ -230,14 +245,17 @@ EOF"
 
 CNIPLUGIN=${CNIPLUGIN:-"bridge"}
 cni_conf_dir=${CNI_CONF_DIR:-"/etc/cni/net.d"}
+cni_conf_parent_dir=${CNI_CONF_PARENT_DIR:-"/etc/cni"}
 cni_bin_dir=${CNI_BIN_DIR:-"/opt/cni/bin"}
 
 if [[ -n "${ARKTOS_NO_CNI_PREINSTALLED}" ]]; then
     echo "CNI plugin will be installed after cluster is started"
     echo "Right now the minimum common cni binary package is being installed only"
+    cleanup_cni
```

Comment:
cleanup_cni only need to be called once before the first if, right?

Reply:
Move function cleanup_cni to be called once prior to line 251.

Reply:
Works for all tests.